11/debian-9/Dockerfile

```diff
@@ -1,28 +1,34 @@
-FROM bitnami/minideb-extras:stretch-r354
+FROM bitnami/minideb-extras-base:stretch-r223
 LABEL maintainer "Bitnami <containers@bitnami.com>"
 
 ENV BITNAMI_PKG_CHMOD="-R g+rwX" \
-    HOME="/"
+    HOME="/" \
+    OS_ARCH="amd64" \
+    OS_FLAVOUR="debian-9" \
+    OS_NAME="linux"
 
 # Install required system packages and dependencies
-RUN install_packages libbsd0 libc6 libedit2 libgcc1 libicu57 liblzma5 libncurses5 libnss-wrapper libssl1.1 libstdc++6 libtinfo5 libxml2 libxslt1.1 zlib1g
-RUN bitnami-pkg unpack postgresql-11.2.0-0 --checksum 9e1a4e02db6b41240d735830b263a1d7190aa55bcf82e664069dc28b4d8ae035
+RUN install_packages libbsd0 libc6 libedit2 libgcc1 libicu57 liblzma5 libncurses5 libnss-wrapper libssl1.1 libstdc++6 libtinfo5 libxml2 libxslt1.1 zlib1g locales
```

Comment:
Why `locales` is required now? Isn't it present at `bitnami/minideb-extras-base`?

Reply:
It is not